custom_components/tesla_custom/update.py

```diff
@@ -27,6 +27,17 @@ async def async_setup_entry(hass: HomeAssistant, config_entry, async_add_entitie
     async_add_entities(entities, True)
 
 
+INSTALLABLE_STATUSES = ["available", "scheduled"]
+
+PRETTY_STATUS_STRINGS = {
```

Comment:
I want to think about this more on whether we should hard code language strings in. I'm ok with this for now.